File: Python/P9.py
import math
import sys

def main():
    for i in range(1, 500):
        for j in range(i, 500):
            if test(i, j):
                print i * j * (1000 - i - j)
                sys.exit

def test(i, j):
    num = 1000 - i - j
    return i ** 2 + j ** 2 == num ** 2

if __name__ == '__main__':
    main()

File: project-euler/java/P9.java
package OneHundred;
/**
 * Write a description of class P9 here.
 * 
 * @author (your name) 
 * @version (a version number or a date)
 */
public class P9
{
    public static void main(String[] args){
        for(double x = 1; x < 1000; x++){
            for(double y = 1; y < 1000; y++){
                double z = Math.sqrt(x*x + y*y);
                if(z+x+y == 1000){
                    System.out.println(x*y*z);
                    System.exit(0);
                }
            }
        }
    }
}
